.circleci/config.yml

```diff
@@ -203,6 +182,16 @@ jobs:
       - store_artifacts:
           path: monitor-ci/test-artifacts/screenshots
           destination: test_artifacts/screenshots
+      - when:
```

Comment:
i'm not sure if putting it as the last step of a job ensures that it it doesn't happen, or if we have to make it a different job, but we can test that easily enough after we merge